specifications/M1/identity-trust-sts-api.yaml

```diff
@@ -0,0 +1,89 @@
+openapi: 3.0.1
+paths:
+  /token:
+    post:
+      operationId: token
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            encoding:
+              access_token:
+                style: form
+              audience:
+                style: form
+              bearer_access_alias:
+                style: form
+              bearer_access_scope:
+                style: form
+              client_id:
+                style: form
+              client_secret:
+                style: form
+              grant_type:
+                style: form
+            schema:
+              type: object
+              properties:
+                access_token:
+                  type: string
+                  description: VP access token to be added as a claim in the SI token
+                audience:
+                  type: string
+                  description: Audience for the SI token
+                bearer_access_alias:
+                  type: string
+                  description: Alias to be use in the sub of the VP access token (default
+                    is audience)
+                bearer_access_scope:
+                  type: string
+                  description: Scope to be added in the VP access token
+                client_id:
+                  type: string
+                  description: Id of the client requesting an SI token
+                client_secret:
+                  type: string
+                  description: Secret of the client requesting an SI token
+                grant_type:
+                  type: string
+                  description: "Type of grant: must be set to client_credentials"
+              required:
```

Comment:
Should the access_token not be required, too?

Reply:
The `access_token` it's not mandatory, 
check [here](https://github.com/eclipse-tractusx/identity-trust/blob/main/specifications/M1/identity.protocol.base.md#6-using-the-oauth-2-client-credential-grant-to-obtain-access-tokens-from-an-sts).